### PR TITLE
should scroll to element if it is hidden by fixed (closes #1057)

### LINKED ISSUE #1057: Click action misses an element if its hidden under fixed element
### Are you requesting a feature or reporting a bug?
bug

### What is the current behavior?
See: https://getbootstrap.com/examples/navbar-fixed-top/. If before click on the "View navbar docs" hyperlink scroll was at the bottom side of the page when testcafe scrolls page to the hyperlink but it leaves hidden under the top navbar and the hyperlink remains unavailable for click.

### What is the expected behavior?
TestCafe should check visibility of clicking element and scroll page to make it visible for click.

### How would you reproduce the current behavior (if this is a bug)?
1. Scroll page to bottom
2. ```await t.click(".btn")```

Tested page URL:
https://getbootstrap.com/examples/navbar-fixed-top/

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
@helen-dikareva yes, I did't know about that issue but this one is definetely part of it

File: test/functional/fixtures/regression/gh-2391/testcafe-fixtures/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Selector } from 'testcafe';
+
+fixture `GH-2391 - should scroll to element if it is hidden by fixed`
```
Comment (AndreyBelym) [inline test/functional/fixtures/regression/gh-2391/testcafe-fixtures/index.js]: Could you please merge this test with the test for GH-1057 issue?

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
Please, don't forget to close this issue with corresponding comment

**Comment by AlexKamaev** [comment]
need to remove skip: 'edge' after the https://github.com/DevExpress/testcafe/issues/2450 is fixed

**Pull request merged**